internal/provider/bootstrap.go

```diff
@@ -198,44 +197,24 @@ func oneTimeBootstrap(l logging.StandardLogger, c *providerConfig.Config, vpnSet
 	}
 
 	if utils.IsOpenRCBased() {
-		// Instead of creating an empty file, let's use the one generated by the k3s installer
-		content, err := os.ReadFile("/etc/init.d/k3s")
-		if err != nil {
-			l.Errorf("Failed to read init file: %s", err.Error())
-			return err
-		}
-
-		if len(content) == 0 {
-			err = fmt.Errorf("empty init file for k3s")
-			l.Errorf("openrc error: %s", err.Error())
-			return err
-		}
-
 		svc, err = openrc.NewService(
 			openrc.WithName(svcName),
-			openrc.WithUnitContent(string(content)),
```

Comment:
this likely will break alpine - in the original openrc service file there is no way to override environment variables (which are actually used to set `K3S_*`)

Reply:
we are not over-writing the init file any longer since we remove the WriteService line. So the original installation unit will be present and reading from the env location

Reply:
yep right, what I'm saying is that we used to write to a custom location, the env file is not read by the original k3s unit (at least at the time of writing the provider)

Reply:
@mudler gotcha, let me validate alpine then, meanwhile, I decided to merge because at least is better to have systemd systems working even if OpenRC ones (alpine) would break